packages/components/tag/src/tag.js

```diff
@@ -231,15 +231,39 @@ const Tag = (props) => {
 };
 
 Tag.propTypes = {
+  /**
+   * Indicates color scheme of the tag.
+   */
   type: PropTypes.oneOf(['normal', 'warning']),
+  /**
+   * Styles object spread onto the tag body..
+   */
   styles: PropTypes.shape({
     body: PropTypes.object,
   }).isRequired,
+  /**
+   * Location the tag links to when enabled.
+   */
   linkTo: PropTypes.string,
+  /**
+   * Disable the tag element along with the option to remove it.
+   */
   isDisabled: PropTypes.bool,
+  /**
+   * Called when remove button is clicked.
+   */
   onRemove: PropTypes.func,
+  /**
+   * Called when tag element is clicked. This is not called with remove button is clicked.
```

Comment:
```suggestion
   * Called when tag element is clicked. This is not called with remove button is clicked.
   * <br>
   * Signature: `(event) => void`
```

For example: https://github.com/commercetools/ui-kit/blob/c20ae492210482697c7c5e181e983e561ff86654/packages/components/data-table/src/data-table.js#L203-L208